examples/hello/subscriber/main.go

```diff
@@ -32,7 +34,15 @@ func main() {
 		md["language"] = *languageFilter
 		log.Printf("Only receiving messages written in %q", *languageFilter)
 	}
-	c, e := s.Start(context.Background(), md)
+
+	// If trying to run basic subscriber you can start like this:
+	// c, e := s.Start(context.Background())
+
+	// If you need a subscriber with a filter, different retention period, or visibility timeout you can use
+	// the following approach:
+	retentionPeriod := 100 * time.Minute   // Length of time a message can stay in the queue
+	visibilityTimeout := 200 * time.Second // Period of time during which subsriber prevents other consumers from receiving and processing the message
```

Comment:
we don't need to define all the different options here -- just specifying the filter would be enough to convey the point. Keeping the example simple and clean is very important!